src/System.Data.SqlClient/tests/ManualTests/SQL/IntegratedAuthenticationTest/IntegratedAuthenticationTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class IntegratedAuthenticationTest
+    {
+        [CheckConnStrSetupFact]
+        public static void IntegratedAuthTest()
+        {
+            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.TcpConnStr);
+            builder.IntegratedSecurity = true;
+            builder.Pooling = false;
+            RunTest(builder.ConnectionString);
+
+            builder.Pooling = true;
+            RunTest(builder.ConnectionString);
+        }
+
+        public static void RunTest(string connectionString)
+        {
+            using (SqlConnection connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+                Console.WriteLine("Connection opened successfully..");
```

Comment:
WriteLine statement is unnecessary, since we either get an exception by this point or success.